### ISSUE TITLE
Cloud intel not fully supports ipv6 address

### ISSUE THREAD

**Issue by MelvinTo** [firewalla/firewalla#437]
```
==> /home/pi/.forever/main.log <==
INFO 2017-10-30 06:31:46 DestIPFoundHook: Found new IP 2604:a880:800:10::182:a001, checking intels...
INFO 2017-10-30 06:31:46 DestIPFoundHook: Found new IP 2a03:b0c0:0:1010::4ed:3001, checking intels...
INFO 2017-10-30 06:31:46 IntelTool: Checking intel for [ '2604:a880:800:10::182:a001' ] [ 'discovery-v6-2.syncthing.net' ]
INFO 2017-10-30 06:31:46 IntelTool: Checking intel for [ '2a03:b0c0:0:1010::4ed:3001' ] [ 'discovery-v6-4.syncthing.net' ]
ERROR 2017-10-30 06:31:47 Bone: **** Error while response HTTP  404
INFO 2017-10-30 06:31:47 Bone: { message: 'Not Found', error: { status: 404 } }
ERROR 2017-10-30 06:31:47 DestIPFoundHook: Failed to process IP 2a03:b0c0:0:1010::4ed:3001, error: 404
ERROR 2017-10-30 06:31:48 Bone: **** Error while response HTTP  404
INFO 2017-10-30 06:31:48 Bone: { message: 'Not Found', error: { status: 404 } }
ERROR 2017-10-30 06:31:48 DestIPFoundHook: Failed to process IP 2604:a880:800:10::182:a001, error: 404
INFO 2017-10-30 06:31:55 DestIPFoundHook: Found new IP 180.163.32.152, checking intels...
INFO 2017-10-30 06:31:55 DestIPFoundHook: Found new IP 2400:6180:0:d0::d9:d001, checking intels...
INFO 2017-10-30 06:31:55 IntelTool: Checking intel for [ '180.163.32.152' ] [ 'masterconn.qq.com' ]
INFO 2017-10-30 06:31:55 IntelTool: Checking intel for [ '2400:6180:0:d0::d9:d001' ] [ 'discovery-v6-3.syncthing.net' ]
ERROR 2017-10-30 06:31:56 Bone: **** Error while response HTTP  404
INFO 2017-10-30 06:31:56 Bone: { message: 'Not Found', error: { status: 404 } }
ERROR 2017-10-30 06:31:56 DestIPFoundHook: Failed to process IP 2400:6180:0:d0::d9:d001, error: 404
ERROR 2017-10-30 06:31:56 Bone: **** Error while response HTTP  404
INFO 2017-10-30 06:31:56 Bone: { message: 'Not Found', error: { status: 404 } }
ERROR 2017-10-30 06:31:56 DestIPFoundHook: Failed to process IP 180.163.32.152, error: 404
INFO 2017-10-30 06:31:58 Main: GC executed, RSS is now 59 MB
INFO 2017-10-30 06:32:01 DestIPFoundHook: Found new IP 64.104.125.228, checking intels...
INFO 2017-10-30 06:32:01 IntelTool: Checking intel for [ '64.104.125.228' ] []
```

**Comment by jerrchen** [comment]
this may not be ipv6, I am seen ipv4 has the same 404 ...

**Comment by MelvinTo** [comment]
fixed in #442
